solidity/contracts/deposit/Deposit.sol

```diff
@@ -385,89 +411,230 @@ contract Deposit is DepositFactoryAuthority {
+        bytes memory _txInputVector,
+        bytes memory _txOutputVector,
+        bytes4 _txLocktime,
+        bytes memory _merkleProof,
+        uint256 _txIndexInBlock,
+        bytes memory _bitcoinHeaders
+    ) public {
+        self.provideRedemptionProof(
+            _txVersion,
+            _txInputVector,
+            _txOutputVector,
+            _txLocktime,
+            _merkleProof,
+            _txIndexInBlock,
+            _bitcoinHeaders
+        );
     }
 
-    /// @notice     Notify the contract that the signers are undercollateralized.
-    /// @dev        Calls out to the system for oracle info.
-    function notifyUndercollateralizedLiquidation() public {
-        self.notifyUndercollateralizedLiquidation();
+//--------------------------- MUTATING HELPERS -------------------------------//
+
+    /// @notice This function can only be called by the deposit factory; use
+    ///         `DepositFactory.createDeposit` to create a new deposit.
+    /// @dev Initializes a new deposit clone with the base state for the
+    ///      deposit.
+    /// @param _tbtcSystem `TBTCSystem` contract. More info in `TBTCSystem`.
+    /// @param _tbtcToken `TBTCToken` contract. More info in TBTCToken`.
+    /// @param _tbtcDepositToken `TBTCDepositToken` (TDT) contract. More info in
+    ///        `TBTCDepositToken`.
+    /// @param _feeRebateToken `FeeRebateToken` (FRT) contract. More info in
+    ///        `FeeRebateToken`.
+    /// @param _vendingMachineAddress `VendingMachine` address. More info in
+    ///        `VendingMachine`.
+    /// @param _m Signing group honesty threshold.
+    /// @param _n Signing group size.
+    /// @param _lotSizeSatoshis The minimum amount of satoshi the funder is
+    ///                         required to send. This is also the amount of
+    ///                         TBTC the TDT holder will be eligible to mint:
+    ///                         (10**7 satoshi == 0.1 BTC == 0.1 TBTC).
+    function initializeDeposit(
+        ITBTCSystem _tbtcSystem,
+        TBTCToken _tbtcToken,
+        IERC721 _tbtcDepositToken,
+        FeeRebateToken _feeRebateToken,
+        address _vendingMachineAddress,
+        uint16 _m,
+        uint16 _n,
+        uint64 _lotSizeSatoshis
+    ) public onlyFactory payable {
+        self.tbtcSystem = _tbtcSystem;
+        self.tbtcToken = _tbtcToken;
+        self.tbtcDepositToken = _tbtcDepositToken;
+        self.feeRebateToken = _feeRebateToken;
+        self.vendingMachineAddress = _vendingMachineAddress;
+        self.initialize(_m, _n, _lotSizeSatoshis);
     }
 
-    /// @notice     Notifies the contract that the courtesy period has elapsed.
-    /// @dev        This is treated as an abort, rather than fraud.
-    function notifyCourtesyTimeout() public {
-        self.notifyCourtesyTimeout();
+    /// @notice This function can only be called by the vending machine.
+    /// @dev Performs the same action as requestRedemption, but transfers
```

Comment:
Feels like `@notice` could use some of that stuff from `@dev` and vice versa, though maybe I'm misinterpreting natspec

Reply:
It's only callable by the vending machine, so arguably `@notice` should have nothing.